README.md

```diff
@@ -1,68 +1,109 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# CAMPUS GAMING NETWORK
 
-## Available Scripts
+Connect with other collegiate gamers for casual or competitive gaming at your school or nearby.
 
-In the project directory, you can run:
+## Getting Started
 
-### `npm start`
+---
```

Comment:
We can remove the `---` in all places because headings like `## Getting Started` already creat a horizontal rule

Reply:
copy that!